docs/LayoutSpecs.json

```diff
@@ -375,7 +375,8 @@
               "path": "DSCore.DateTime.TimeOfDay"
             },
             {
-              "path": "DSCore.DayOfWeek"
+              "path": "DSCore.DayOfWeek",
+              "pathIcon": "/src/resources/icons/DSCore.DayOfWeek.png"
```

Comment:
@Benglin If the icon url is unspecified, it is constructed using the name of the path. Since we're not adding 'src/resources/icons/' and '.png' extensions anymore, the resulting icon url is http://localhost:3456/DSCore.DayOfWeek. This only applies to nodes in the 'none' category.